Sources/Shared/API/Webhook/Sensors/WatchBatterySensor.swift

```diff
@@ -15,16 +15,24 @@ final class WatchBatterySensor: SensorProvider {
         case .paired:
             let batteryDecimal = Communicator.shared.mostRecentlyReceievedContext
                 .content[WatchContext.watchBattery.rawValue] as? Float ?? -1
+            let batteryState = Communicator.shared.mostRecentlyReceievedContext
+                .content[WatchContext.watchBatteryState.rawValue] as? DeviceBattery.State ?? .unplugged
             var battery = batteryDecimal > -1 ? Int(batteryDecimal * 100) : -1
-            let icon: String = BatteryIcon.forBatteryLevel(battery, state: .unplugged)
+            let icon: String = BatteryIcon.forBatteryLevel(battery, state: batteryState)
             sensors.append(WebhookSensor(
-                name: "Watch Battery",
-                uniqueID: "watch-battery",
+                name: "Watch Battery Level",
+                uniqueID: "watch-battery_level",
                 icon: icon,
                 deviceClass: .battery,
                 state: battery,
                 unit: "%"
             ))
+            sensors.append(WebhookSensor(
+                name: "Watch Battery State",
+                uniqueID: "watch-battery_state",
```

Comment:
Let's use just dashes here, so:
```suggestion
                uniqueID: "watch-battery-state",
```